apps/sumo-aws-apps/cloudtrail/TestCfTemplate.sh

```diff
@@ -1,51 +1,79 @@
 #!/bin/bash
 
-export AWS_REGION="us-east-1"
-export AWS_PROFILE="personal"
+export AWS_REGION="us-east-2"
+export AWS_PROFILE="default"
 # App to test
 export AppTemplateName="cloudtrail"
 export AppName="cloudtrail"
-export InstallTypes=("onlycisfoundationapp")
+export InstallTypes=("all")
 
 for InstallType in "${InstallTypes[@]}"
 do
     export CloudTrailLogsBucketName="${AppName}-${InstallType}-qwerty"
-    export QSS3BucketName="sumologiclambdahelper"
-    export QSS3BucketRegion="us-east-1"
+    export QSS3BucketName="aws-quickstart"
```

Comment:
Replace the aws-quickstart bucket with a new one. As discussed we don't have permission to upload.